Source/relay/TourGuide/Support/IOTMs.ash

```diff
@@ -82,6 +84,9 @@ void initialiseIOTMsUsable()
         __iotms_usable[lookupItem("Distant Woods Getaway Brochure")] = true;
     if (lookupItem("Eight Days a Week Pill Keeper").available_amount() > 0)
         __iotms_usable[lookupItem("Eight Days a Week Pill Keeper")] = true;
+	if (lookupItem("cosmic bowling ball").available_amount() > 0)
```

Comment:
my one concern here is that this means all tiles referencing CBB (specifically the banish tile) will stop listing it when it's out of your inventory. seems like bad news that may make some folks forget they have that banish accessible. :-(

Reply:
(resolving the other comment as when i got to it the other comment became immaterial)